src/test/groovy/com/ullink/MSBuildTaskBuilder.groovy

```diff
@@ -33,12 +33,12 @@ class MSBuildTaskBuilder {
     }
 
     public MSBuildTaskBuilder withArtifact(String artifactPath) {
-        artifacts.add(new File(artifactPath))
+        artifacts.add(new File(artifactPath.replace('\\', File.separator)))
```

Comment:
I agree but it's for unit tests. We could find a more global solution when such think will be used.